ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```diff
@@ -39,6 +40,8 @@ spec:
           value: $(params.git_fork_url)
         - name: revision
           value: $(params.git_pr_branch)
+        - name: depth
+          value: 25
```

Comment:
Where does this number come from?

Reply:
This is because we need to check out the base commit from the repo and by default the git-clone task makes a shallow clone of depth 1 (from the head commit). 25 should be high enough to include the base commit in almost all cases.